hw/xilinx/Makefile

```diff
@@ -13,10 +13,30 @@ XILINX_PROJECT_REPORTS_DIR ?= ${XILINX_PROJECT_BUILD_DIR}/reports
 
 # List of the Xilinx IPs to build and import in the design
 # Parsing from directory ips/
-XILINX_IP_LIST = $(shell basename --multiple ${XILINX_IPS_ROOT}/xlnx_*)
+
+XILINX_COMMON_IP_LIST = $(shell basename --multiple ${XILINX_IPS_ROOT}/common/xlnx_*)
+XILINX_HPC_IP_LIST = $(shell basename --multiple ${XILINX_IPS_ROOT}/hpc/xlnx_*) 
+XILINX_EMBEDDED_IP_LIST = $(shell basename --multiple ${XILINX_IPS_ROOT}/embedded/xlnx_*) 
+
+XILINX_IP_LIST = ${XILINX_COMMON_IP_LIST}
 
 # List of IPs' xci files
-XILINX_IP_LIST_XCI := $(foreach ip,${XILINX_IP_LIST},${XILINX_IPS_ROOT}/${ip}/build/${ip}.srcs/sources_1/ip/${ip}/${ip}.xci)
+XILINX_COMMON_IP_LIST_XCI := $(foreach ip,${XILINX_COMMON_IP_LIST},${XILINX_IPS_ROOT}/common/${ip}/build/${ip}.srcs/sources_1/ip/${ip}/${ip}.xci)
```

Comment:
this should be reflected in [hw/xilinx/README.md](https://github.com/MaistoV/UninaSoC/blob/main/hw/xilinx/README.md?plain=1#L2) too